linux/python_tools.bsh

```diff
@@ -96,3 +96,26 @@ function _array_to_python()
 
   dynamic_set "${__array_to_python_indirect}" "${__array_to_python_string}"
 }
+
+#**
+# .. function:: vsi::python::virtualenv_bin
+#
+# :Arguments: * ``$1`` - Virtual environment path, do not include ``bin`` or ``Scripts``
+#             * ``$2`` - Name of binary to run, without ``.exe`` extension
+#
+# A cross-OS compatible function to call a binary in a virtual environment folder that handles the differences between Windows and Linux.
+#
+# .. rubric:: Example
+#
+# .. code-block:: bash
+#
+#   "$(vsi::python::virtualenv_bin "${my_python_dir}" pip)" install numpy
+#**
+function vsi::python::virtualenv_bin()
```

Comment:
should this be called virtualenv_run?

Reply:
It can also be used for checking to see if the files exists. The `bin` refers to the `bin` directory in Linux.